### ISSUE TITLE
Better upload / processing feedback?

### ISSUE THREAD

**Issue by PeterTonoli** [sgsinclair/VoyantServer#29]
On large corpus, irrespective of what Voyant Server is doing, the status is 'Uploading'.
Perhaps a bar graph, or status bar of the upload, and then a status change to 'processing', or similar, would give better feedback to users as to where things were in the upload/processing steps?

**Comment by sgsinclair** [comment]
Yes, definitely. In fact, there's already a framework in place both client and server side for progress monitoring, we just haven't yet refactored corpus creation to use it (it's a somewhat big job since corpus creation happens in a variety of ways in the larger family of Voyant interfaces). It's high on the docket for this summer.

**Comment by PeterTonoli** [comment]
👍
